rosdep/base.yaml

```diff
@@ -4813,6 +4813,19 @@ yasm:
   fedora: [yasm]
   gentoo: [dev-lang/yasm]
   ubuntu: [yasm]
+yamllint:
+  arch: [yamllint]
+  debian:
+    buster: [yamllint]
+    sid: [yamllint]
+    stretch: [yamllint]
+  fedora: [yamllint]
+  gentoo: [yamllint]
+  ubuntu:
+    xenial: [yamllint]
+    zesty: [yamllint]
```

Comment:
Please add yakkety in the middle.

Reply:
Added.

[Package's page for yakkety](https://packages.ubuntu.com/en/yakkety/yamllint) doesn't seem to exist but I assume it's due to its EoL status.